doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -7,220 +7,125 @@ NumPy for MATLAB users
-      initializers**. As in, ``array([[1,2,3],[4,5,6]])``.
-   -  The ``matrix`` constructor additionally **takes a convenient
-      string initializer**. As in ``matrix("[1 2 3; 4 5 6]")``.
-
-There are pros and cons to using both:
-
--  ``array``
-
-   -  ``:)`` Element-wise multiplication is easy: ``A*B``.
-   -  ``:(`` You have to remember that matrix multiplication has its own
-      operator, ``@``.
-   -  ``:)`` You can treat one-dimensional arrays as *either* row or column
-      vectors. ``A @ v`` treats ``v`` as a column vector, while
-      ``v @ A`` treats ``v`` as a row vector. This can save you having to
-      type a lot of transposes.
-   -  ``:)`` ``array`` is the "default" NumPy type, so it gets the most
-      testing, and is the type most likely to be returned by 3rd party
-      code that uses NumPy.
-   -  ``:)`` Is quite at home handling data of any number of dimensions.
-   -  ``:)`` Closer in semantics to tensor algebra, if you are familiar
-      with that.
-   -  ``:)`` *All* operations (``*``, ``/``, ``+``, ``-`` etc.) are
-      element-wise.
-   -  ``:(`` Sparse matrices from ``scipy.sparse`` do not interact as well
-      with arrays.
-
--  ``matrix``
-
-   -  ``:\\`` Behavior is more like that of MATLAB matrices.
-   -  ``<:(`` Maximum of two-dimensional. To hold three-dimensional data you
-      need ``array`` or perhaps a Python list of ``matrix``.
-   -  ``<:(`` Minimum of two-dimensional. You cannot have vectors. They must be
-      cast as single-column or single-row matrices.
-   -  ``<:(`` Since ``array`` is the default in NumPy, some functions may
-      return an ``array`` even if you give them a ``matrix`` as an
-      argument. This shouldn't happen with NumPy functions (if it does
-      it's a bug), but 3rd party code based on NumPy may not honor type
-      preservation like NumPy does.
-   -  ``:)`` ``A*B`` is matrix multiplication, so it looks just like you write
-      it in linear algebra (For Python >= 3.5 plain arrays have the same
-      convenience with the ``@`` operator).
-   -  ``<:(`` Element-wise multiplication requires calling a function,
-      ``multiply(A,B)``.
-   -  ``<:(`` The use of operator overloading is a bit illogical: ``*``
-      does not work element-wise but ``/`` does.
-   -  Interaction with ``scipy.sparse`` is a bit cleaner.
-
-The ``array`` is thus much more advisable to use.  Indeed, we intend to
-deprecate ``matrix`` eventually.
 
-Table of Rough MATLAB-NumPy Equivalents
+Rough equivalents
 =======================================
 
 The table below gives rough equivalents for some common MATLAB
-expressions. **These are not exact equivalents**, but rather should be
-taken as hints to get you going in the right direction. For more detail
-read the built-in documentation on the NumPy functions.
+expressions. These are similar expressions, not equivalents. For
+details, see the `documentation
+<https://numpy.org/doc/stable/reference/routines.html>`_. 
```

Comment:
This should be an internal Sphinx. I don't remember the exact syntax...

Reply:
So the Routines index.html start with reference:
`.. _reference:`
But using this `documentation <reference>` can't find that index page. I'm having a bit of trouble figuring out the rst cross-refs. 

@bjnath and @melissawm do you know how to get that cross-ref?